tests/factories.py

```diff
@@ -436,6 +437,22 @@ def write_request_file(
         review_file(request, path, UserFileReviewStatus.REJECTED, *checkers)
 
 
+def create_request_file_bad(request_file, bad_path):
```

Comment:
Could we name this `create_request_file_bad_path` so it's obvious what's bad about it?

Reply:
yep done